atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -412,18 +413,19 @@ SerializableTransactionManager serializable() {
     }
 
     private QosClient getQosClient(Optional<ServiceConfiguration> serviceConfiguration) {
-        return serviceConfiguration.map(this::createAtlasDbQosClient).orElseGet(FakeQosClient::getDefault);
+        return serviceConfiguration.map(this::createAtlasDbQosClient).orElse(FakeQosClient.INSTANCE);
     }
 
     private QosClient createAtlasDbQosClient(ServiceConfiguration serviceConfiguration) {
         QosService qosService = JaxRsClient.create(QosService.class,
                 userAgent(),
                 ClientConfigurations.of(serviceConfiguration));
+        // TODO(nziebart): create a RefreshingRateLimiter
         ScheduledExecutorService scheduler = new InstrumentedScheduledExecutorService(
                 Executors.newSingleThreadScheduledExecutor(),
                 AtlasDbMetrics.getMetricRegistry(),
                 "qos-client-executor");
-        return new AtlasDbQosClient(qosService, scheduler, config().getNamespaceString(), QosRateLimiter.create());
+        return new AtlasDbQosClient(QosRateLimiter.create(), new QosMetrics());
```

Comment:
looks like `qosService` and `scheduler` are now unused (although I suspect there's some remoting magic going on here?)

Reply:
these are removed in a subsequent PR